# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/script/ScriptLoader.h
# Commit: 85fd9fc9cffa
# Full Hash: 85fd9fc9cffa98039328796d30a2d10386283b3e
# Author: Yulia Startsev <ystartsev@mozilla.com>
# Date: 2022-02-21 21:39:46
# Regressor Bug: 1742437
# File Overlap Count: 2
# Description:
#   Bug 1742437 - Move module loader files into a central folder; r=jonco
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D137532
# ==============================================================================

diff -r 7ecc4f9b20ad -r 85fd9fc9cffa dom/script/ScriptLoader.h
--- a/dom/script/ScriptLoader.h	Mon Feb 21 15:39:08 2022 +0000
+++ b/dom/script/ScriptLoader.h	Mon Feb 21 15:39:08 2022 +0000
@@ -56,14 +56,18 @@
 class AutoJSAPI;
 class DocGroup;
 class Document;
-class ModuleLoader;
 class LoadedScript;
 class ScriptLoaderInterface;
+class ModuleLoader;
 class ModuleLoadRequest;
 class ModuleScript;
 class SRICheckDataVerifier;
 class SRIMetadata;
 class ScriptLoadHandler;
+class ScriptLoadRequest;
+class ScriptLoadRequestList;
+class ScriptLoadContext;
+
 class ScriptLoader;
 class ScriptRequestProcessor;
 